tests/integ/test_tf_script_mode.py

```diff
@@ -131,7 +131,9 @@ def test_mnist_distributed(sagemaker_session, instance_type, tf_full_version):
         train_instance_count=2,
         train_instance_type=instance_type,
         sagemaker_session=sagemaker_session,
-        py_version=tests.integ.PYTHON_VERSION,
+        py_version="py37"
```

Comment:
what about other places in this file where `py_veresion=tests.integ.PYTHON_VERSION`?

Reply:
I was going to update them all once the container is released. But probably it's better to update them all at once.

Reply:
hmm...maybe it would be cleaner to put this into a fixture? i.e.

```python
@pytest.fixture
def py_version():
    return "py37" if tf_full_version == TensorFlow._LATEST_1X_VERSION else tests.integ.PYTHON_VERSION
```

and then just use `py_version` in all the tests

Reply:
👍 Yes. This would be much better.